cmake/stm32/mp1.cmake

```
@@ -0,0 +1,31 @@
+set(STM32_MP1_TYPES 
+    MP151Axx MP151Cxx
+    MP153Axx MP153Cxx 
+    MP157Axx MP157Cxx)
+
+set(STM32_MP1_TYPE_MATCH
+    "MP151[AD]A.?" "MP151[CF]A.?"  

```

I have a slight preference for this :
```suggestion
    "MP151[AD]..?" "MP151[CF]..?"  
```
But regarding previous comment it might be better to leave as is 👍